api/src/main/java/org/openmrs/validator/PersonAddressValidator.java

```diff
@@ -63,12 +63,21 @@ public void validate(Object object, Errors errors) {
 			log.debug(this.getClass().getName() + ".validate...");
 		}
 		
-		if (object == null) {
+		/*if (object == null) {
 			throw new IllegalArgumentException("The personAddress object should not be null");
-		}
+		}*/
 		
 		PersonAddress personAddress = (PersonAddress) object;
-		
+		try {
+			// Validate that the person Address  object is not null
+			if (personAddress == null) {
```

Comment:
I am not sure if a rejection would lead to the following code not being executed. 
You probably should make the function execution stop if the value is null. Otherwise we may get a Null Value exception at Line 83